src/_data/env.js

```diff
@@ -0,0 +1,5 @@
+require('dotenv').config();
+
+module.exports = {
+  environment: process.env.ELEVENTY_ENV || 'development'
```

Comment:
This would need to be set to something other than 'development' over on Netlify… 